### ISSUE TITLE
"Full-zoom" with "move by dragging" not working as expected

### ISSUE THREAD

**Issue by deus-e-x** [TheFantasticWarrior/chrome-extension-imagus#63]
Using the original Imagus plugin, with the "Full-zoom" default mode set to "move by dragging":

1. Right-click on the pop-up image once to make it remain on-screen.
2. Drag the image by holding down the left mouse button on the image, the image is visible during the dragging process.
3. Upon releasing the left mouse button the image remains on-screen at its current position.
4. You may either drag the image again by holding down the left mouse button on the image or left-click outside the image to close the pop-up image.

Using the Imagus-mod plugin, with the "Full-zoom" default mode set to "move by dragging":

1. Right-click on the pop-up image once to make it remain on-screen.
2. Trying to drag the image by holding down the left mouse button on the image nothing happens. But when holding down the left mouse button on the image AND quickly moving the mouse cursor slightly you get to see some error symbol (strike-through circle). Now upon releasing the left mouse button, the pop-up image automatically follows the mouse cursor.
4. Left-clicking the image will close it, right-clicking the image makes it remain on-screen again.

This appears to be a bug as Imagus-mod unexpectedly does not recreate the original behavior of Imagus.

![Screenshot](https://github.com/TheFantasticWarrior/chrome-extension-imagus/assets/45689656/3dc91d79-f1e8-43bc-ad95-d32569ca6c0f)

**Comment by TheFantasticWarrior** [comment]
Is this v0.10.16 only? I had to change some things to make the zoom in/out work without entering full zoom, and the original author had a lot complicated checks going on and I deleted some without knowing what some of the checks do

**Comment by deus-e-x** [comment]
No, the issue is also apparent with v0.10.15, which marks the first version of your fork that I used. Currently, this is a serious showstopper for me, preventing me from using it regularly.

I hope you can fix it. Feel free to ask me to assist/test whenever required.

**Comment by yarub123** [comment]
Yeah I'm still getting this as well 😞. Nice to see at least someone brought it up! Thanks for the awesome mod brothaman. Appreciate you @TheFantasticWarrior
